README.md

```diff
@@ -1 +1,11 @@
-# hw01
\ No newline at end of file
+# hw01 
+Yixi Chen
+
+### Yixi's Bio
+
+Yixi is originally from China. She graduated from **Mount Holyoke College** with double major in **History and Economics**. After graduation, she worked at Mastercard as an analyst for three years. Her top 3 travel destinations are:
+1. Granada, Spain
+2. Kyoto, Japan
+3. Jeruselum, Isareal  
+[Yixi's LinkedIn Page](https://www.linkedin.com/in/yixi-chen-7b42bb63/)
+![My photo at UChicago](Photo.jpg)
```

Comment:
If you want to put the link to your LinkedIn page and your photo in different rows, I think you need to use # to start a new paragraph.

Reply:
Actually you just need a line break

```
Yixi is originally from China. She graduated from **Mount Holyoke College** with double major in **History and Economics**. After graduation, she worked at Mastercard as an analyst for three years. Her top 3 travel destinations are:

1. Granada, Spain
2. Kyoto, Japan
3. Jeruselum, Isareal  

[Yixi's LinkedIn Page](https://www.linkedin.com/in/yixi-chen-7b42bb63/)

![My photo at UChicago](Photo.jpg)
```

Yixi is originally from China. She graduated from **Mount Holyoke College** with double major in **History and Economics**. After graduation, she worked at Mastercard as an analyst for three years. Her top 3 travel destinations are:

1. Granada, Spain
2. Kyoto, Japan
3. Jeruselum, Isareal  

[Yixi's LinkedIn Page](https://www.linkedin.com/in/yixi-chen-7b42bb63/)

![My photo at UChicago](Photo.jpg)